solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -3,103 +3,103 @@
 import json
 
 ATM_FILE_PLACE = 1
-COSTUMER_CHOICE_OPTIONS = "1 - View balance \n2 - Withdraw \n3 - Deposit " \
-                          "\n4 - Change password \n5 - Exit\n"
-
-costumer_id = ""
-costumer_password = ""
+BALANCE_OF_COSTUMER = 'balance'
+PASSWORD_OF_COSTUMER = 'password'
+COSTUMER_CHOICE_OPTIONS = "1 - View balance \n" \
+                          "2 - Withdraw \n" \
+                          "3 - Deposit \n" \
+                          "4 - Change password \n" \
+                          "5 - Exit\n"
+SWITCHER = {
+    '1': lambda data, identifier: print(data[identifier][
+                                            BALANCE_OF_COSTUMER]),
+    '2': lambda data, identifier: cash_withdraw(data, identifier),
+    '3': lambda data, identifier: cash_deposit(data, identifier),
+    '4': lambda data, identifier: change_password(data, identifier)
+}
```

Comment:
In general we don't like lambdas and prefer you just use helper functions. Its easier to read the code that way.

for 2-4 the lambdas are completely unnecessary as you can simply use the function without any arguments as the value.

Here is an example of what I mean:
def f_1(a,b):
    print(a+b)

def f_2(a,b):
    print(a-b)

MATCHER = {1:f_1, 2:f_2}
result_1 = MATCHER[1](1,2) # equals 3

result_2 = MATCHER[2](1,2) #equals -1

print(result_1, result_2)